gulpfile.js

```diff
@@ -69,6 +69,7 @@ gulp.task('release', ['build-css', 'merge-datasources', 'release-app', 'release-
 
 gulp.task('watch-app', ['prepare'], function() {
     return watch(appJSName, appEntryJSName, false);
+    // TODO: make this automatically trigger when ./lib/Views/*.html get updated
```

Comment:
Hmm I just tried it.  I changed RelatedMaps.html, saved it, and my `gulp watch-app` noticed that change and rebuilt nationalmap.js.  Maybe I don't understand what you're referring to?